serveradmin/servershell/static/js/servershell/result.js

```diff
@@ -59,8 +59,8 @@ get_row_html = function(object, number) {
     }
 
     // Standard columns which should always be present
-    row.append(`<td><input tabindex="3" type="checkbox" name="server" value="${object.object_id}"/></td>`);
-    row.append(`<td>${number + servershell.offset}</td>`);
+    row.append($('<td>').append($('<input tabindex="3" type="checkbox" name="server"/>').val(object.object_id)));
```

Comment:
If that is possible like that, then it looks awesome! 👍🏼 
I'm just not sure if in the end just the `input` gets appended to the row and not the `td`. Did you test it already on staging?

Reply:
I double checked it - it works.

![Screenshot from 2021-04-08 10-31-13](https://user-images.githubusercontent.com/3993306/113994448-af8fa180-9855-11eb-8451-4bfd0a608c85.png)

Reply:
Awesome! 👍🏼